src/templates/app/user/authenticate.jsx

```diff
@@ -252,12 +261,19 @@ const Authenticate = () => (
                             <img className='gr-padding-20' src={it.url_for('images/pages/authenticate/valid.svg')} />
                             <h1 className='gr-padding-10'>{it.L('Your proof of identity has been verified successfully')}</h1>
                             <p id='text_verified_poa_required' className='invisible'>{it.L('You must also submit a proof of address.')}</p>
+                            <p id='text_verified_edd_required' className='invisible'>{it.L('You must also submit a proof of income.')}</p>
```

Comment:
I suppose , proof of income doesn’t connect to others features. So maybe we don’t need this message. Correct me please, if I’m not right